joint_names: &aria_v2_joints
    - Joint_1
    - Joint_2
    - Joint_3
    - Joint_4
    - Joint_5
    - Joint_6
    - Joint_7

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

joint_trajectory_controller:
  type: position_controllers/JointTrajectoryController
  joints: *aria_v2_joints

  # This is probably required for the real hardware
  # -----------------------------------------------
  #  constraints:
  #    goal_time: 5.0
  #    #      stopped_velocity_tolerance: 0.0
  #    Joint_1: {trajectory: 0.0, goal: 0.0}
  #    Joint_2: {trajectory: 0.0, goal: 0.0}
  #    Joint_3: {trajectory: 0.0, goal: 0.0}
  #    Joint_4: {trajectory: 0.0, goal: 0.0}
  #    Joint_5: {trajectory: 0.0, goal: 0.0}
  #    Joint_6: {trajectory: 0.0, goal: 0.0}
  #    gains:
  #      Joint_1: {p: 10.0,  i: 1.0, d: 0.01, i_clamp: 1}
  #      Joint_2: {p: 10.0,  i: 1.0, d: 0.01, i_clamp: 1}
  #      Joint_3: {p: 10.0,  i: 1.0, d: 0.01, i_clamp: 1}
  #      Joint_4: {p: 10.0,  i: 1.0, d: 0.01, i_clamp: 1}
  #      Joint_5: {p: 10.0,  i: 1.0, d: 0.01, i_clamp: 1}
  #      Joint_6: {p: 10.0,  i: 1.0, d: 0.01, i_clamp: 1}

gripper_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - Joint_7
  constraints:
    goal_time: 5.0
    Joint_7: {trajectory: 0.0, goal: 0.0}
    gains:
      Joint_7: {p: 10.0,  i: 1.0, d: 0.01, i_clamp: 1}
  allow_partial_joints_goal: true
